cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -19,6 +19,7 @@ import           Prelude (String)
 
 import           Data.Aeson (ToJSON (..), (.=))
 import qualified Data.Aeson as Aeson
+import qualified Data.Aeson as J
```

Comment:
Is the qualified import necessary? We usually qualify as `Aeson`. I think we're ok to just use `toJSON` here.

Reply:
Oops.  I missed that.  Fixed!